test/plugin/scenarios/spring-scheduled-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+  - serviceName: spring-scheduled-scenario
+    segmentSize: ge 2
+    segments:
```

Comment:
From my reading of your test case codes, there should be a segment starting with `SchedulingJob#work`, where it is? Or do I miss anything?

Reply:
Yes, it's on [line 37](https://github.com/apache/skywalking/pull/5339/files/7f6bba6a27a0ba0ccb41c639f0002e5276b7de3a#diff-f1c217f8303ef2da8c91cd14aaa67300R37).

Reply:
You missed the segment from the peer. There should be entry span representing the server side of this exit span.

I know it is not the thing you added, but it should work, and the segment is expected.